extra/src/main/java/aima/extra/probability/LogProbabilityNumber.java

```diff
@@ -28,24 +28,18 @@
 	 * According to the IEEE 754 format, double values have a precision of 15.95
 	 * decimal digits. Here, max_precision is set to 15 digits by default.
 	 */
-	private static final Integer DEFAULT_MAX_PRECISION = MathContext.DECIMAL64.getPrecision() - 1;
+	private static Integer MAX_PRECISION = MathContext.DECIMAL64.getPrecision() - 1;
 
 	/**
 	 * RoundingMode.HALF_EVEN statistically minimizes cumulative error when
 	 * applied repeatedly over a sequence of calculations.
 	 */
-	private static final RoundingMode DEFAULT_PRECISION_ROUNDING_MODE = RoundingMode.HALF_EVEN;
+	private static RoundingMode ROUNDING_MODE = RoundingMode.HALF_EVEN;
 
```

Comment:
As with DoubleProbabilityNumber I think these two statics may need to become instance based.